fdbserver/storageserver.actor.cpp

```diff
@@ -1176,13 +1176,20 @@ ACTOR Future<Version> waitForVersionActor(StorageServer* data, Version version,
 // If the latest commit version that mutated the shard(s) being served by the specified storage
 // server is below the client specified read version then do a read at the latest commit version
 // of the storage server.
-Version getRealReadVersion(VersionVector& ssLatestCommitVersions, Tag& tag, Version specifiedReadVersion)
-{
-	Version realReadVersion = ssLatestCommitVersions.hasVersion(tag) ? ssLatestCommitVersions.getVersion(tag) : specifiedReadVersion;
+Version getRealReadVersion(VersionVector& ssLatestCommitVersions, Tag& tag, Version specifiedReadVersion) {
```

Comment:
Maybe define as a member function of `VersionVector`.

Reply:
Actually, we may not need this method. So let's leave it here for now.

Reply:
The method in VersionVector asserts if the tag is not in the vector, so the caller must check if the tag is in the vector or not? A method that returned a status rather than asserting might be a nice way to reduce caller burden.